src/promptflow/tests/sdk_cli_azure_test/conftest.py

```diff
@@ -12,14 +12,9 @@
 
 import jwt
 import pytest
-from azure.ai.ml import MLClient
```

Comment:
why do we need to do this?

Reply:
this conftest is loaded during test scanning. This is fix for your VSCode/PyCharm.

Or else they cannot find your tests because azure.ai.ml is not installed.